src/ActionModal/components/UndelegationModal.vue

```diff
@@ -38,7 +50,12 @@
     >
       <TmField
         id="from"
-        :value="sourceValidator | validatorEntry"
+        :value="
+          // prettier-ignore
```

Comment:
?

Reply:
it is also outdated. No it is a computed, this one:

```
    enhancedSourceValidator() {
      return validatorEntry(this.sourceValidator)
    },
```

The reason is because, for some reason, `validatorEntry` was throwing an awful error, don't know why. This typical error "component has not being loaded but is mentioned during rendered. Check your reactive component".

Could only fix it by doing this 🤷